src/routes/tools/slr-coastal-inundation/_Map/TileIndexes.svelte

```diff
@@ -23,7 +23,7 @@
   });
 
   const mapLayersProps = ({ id, checked, color }) => ({
-    id,
+    id: `${id}-tile-index`,
```

Comment:
Not really needed now that the raster layers are using unique ids, but it helps for debugging.